admin_guide/managing_networking.adoc

```diff
@@ -122,6 +122,29 @@ admissionConfig:
 
 [[admin-guide-controlling-egress-traffic]]
 == Controlling Egress Traffic
+
+As a cluster administrator you can allocate a number of static IP addresses to
+a specific node at the host level. If an application developer needs an
```

Comment:
Maybe s/an unchanging source IP/a dedicated IP address

This is a better common term.